package.json

```diff
@@ -23,8 +22,10 @@
     "test": "tests"
   },
   "scripts": {
+    "prepare": "tsc -b ./tsconfig.json",
+    "clean": "git clean -x -f",
```

Comment:
Just to confirm, this does **not** delete `node_modules` right?

Reply:
Confirmed it does not